iclaims_test.go

```diff
@@ -174,11 +174,11 @@ func Test_IClaims_SetBootSeed_invalid(t *testing.T) {
 func Test_IClaims_SetCertificationReference_invalid(t *testing.T) {
 	tv := makeIClaims(t)
 
-	expectedErr := `wrong syntax for claim: MUST be in EAN-13 format`
+	expectedErr := `wrong syntax for claim: MUST be in EAN-13`
 
 	for _, c := range tv {
 		err := c.SetCertificationReference("")
-		assert.EqualError(t, err, expectedErr)
+		assert.ErrorContains(t, err, expectedErr)
```

Comment:
A better test for invalid Certification Reference is to have some Invalid value like "XYZ", rather than a non-existent value?

Reply:
why is it better?  we just need an input that doesn't match the regex -- of which there are quasi-infinite :-)

Reply:
Well this is a test of No Certification Reference, isn't it? The test name says invalid, hence the confusion

Reply:
a test for no certification reference would be one that doesn't call SetCertificationReference, not one that calls it with an invalid value.

Reply:
Well I meant nil CertificateReference passed is a testNilCertificateReference 
Usually when we do not want to set any value to an element, of type string we set it to empty( "") string.

Reply:
> Well I meant nil CertificateReference passed is a testNilCertificateReference
> Usually when we do not want to set any value to an element, of type string we set it to empty( "") string.

The semantics of the method is: you either pass me a CR in the correct form (i.e. EAN-13 or EAN-13+5) or I won't set it (CR is an optional claim).